validator/testcases/content.py

```diff
@@ -157,7 +157,7 @@ def test_packed_packages(err, xpi_package=None):
                 err.notice(
                     err_id=('testcases_content',
                             'test_packed_packages',
-                            'blacklisted_js_library'),
+                            'denied_js_library'),
```

Comment:
⚠️ We explicitly check for this exact string in https://github.com/mozilla/addons-server/blob/master/static/js/zamboni/files.js#L524 to update the validation report and file viewer.